templates/flasher.lua

```diff
@@ -522,6 +524,11 @@ function M.detect(tPlugin, aAttr, tBus, ulUnit, ulChipSelect, fnCallbackMessage,
 		local ulMmioConfiguration = atParameter.ulMmioConfiguration
 		ulMmioConfiguration = ulMmioConfiguration or 0xffffffff
 
+		-- Set flag for use of SFDP erase operation if smart_erse command was used
+		if bUseSfdpErase then
+			ulFlags = ulFlags + 1
+		end
+
```

Comment:
Define a constant, like
FLAG_DETECT_SPI_USE_SFDP_ERASE = 1.
Also, with future extensions in mind, we might take a parameter ulFlags rather than having a separate parameter for every setting.